oauth-implicit.html.md.erb

```diff
@@ -0,0 +1,62 @@
+---
+title: OAuth Implicit 
+owner: API
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> OAuth Implicit Tutorial
+
+**Note: The Swisscom OAuth is only usable if it is approved and granted from the API team - contact us via the [contact form](https://developer.swisscom.com/support)**
+
+This tutorial describes an easy way to get an OAuth authorization code & access token and shows the integration flow.
+
+Tools:
+
+* Google Chrome
+* Modify Headers Plugin for google Chrome
+* Localhost (How to setup a localhost on Windows)
+
+### 1. Get OAuth Implicit Token
+
+Open this URL in your browser - you need a valid client id.
+```
+https://consent.swisscom.com/c/oauth2/auth?response_type=token&redirect_uri=http%3A%2F%2Flocalhost&client_id=%YOUR_CLIENT_ID%
+```
+
+In this example http%3A%2F%2Flocalhost is used as redirect URL (URL encoded).
+
+![Implicit Consent URL](img/implicit_consent_url.png)
+
+After hitting the enter key the Swisscom Login page appears.
+
+#### 2. Login with given Swisscom credentials
+
+The user has to login with his Swisscom Credentials (username & password).
+
+![Swisscom login page](img/scloginstep2en.png)
+
+#### 3. Consent Page - Authorise
+
+After login successfully a consent page appears with a request to agree that this page/app has the permission to read the users information.
```

Comment:
"After successful login..." or "After logging in successfully..." (also: is there an unsuccessful login?)